### PR TITLE
[Components with more than 1 service] LPS-202951 Refactor JournalContentImpl component

### PR DESCRIPTION
**Pull request by rcsdnj**
JournalContentImpl component was being registered for CTEventListener.class and IdentifiableOSGiService.class. With this refactor, CTEventListener interface handling was moved to another class (JournalContentEventListenerImpl) and IdentifiableOSGiService.class was registered in a later moment (on component's activation)

### CODE REVIEW

**Comment by rcsdnj** [comment]
ci:test:sf

**Comment by rcsdnj** [comment]
ci:test:relevant

**Comment by rcsdnj** [comment]
ci:test:sf

File: modules/apps/journal/journal-service/src/main/java/com/liferay/journal/internal/util/JournalContentEventListenerImpl.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Roberto Cassio Silva do Nascimento Junior
+ */
+@Component(service = CTEventListener.class)
+public class JournalContentEventListenerImpl implements CTEventListener {
+
+	@Override
+	public void onAfterPublish(long ctCollectionId) {
```
Comment (rafaprax) [inline modules/apps/journal/journal-service/src/main/java/com/liferay/journal/internal/util/JournalContentEventListenerImpl.java:19]: Inline JournalContentEventListenerImpl into the JournalContentImpl.java and register the service manually.

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by rafaprax** [comment]
sent here https://github.com/liferay-core-infra/liferay-portal/pull/6407

**Comment by rafaprax** [comment]
ci:help

**Pull request closed without merging**